includes/class-wc-subscriptions-core-plugin.php

```diff
@@ -54,7 +54,7 @@ public function __construct( $autoloader = null ) {
 		if ( $autoloader ) {
 			$this->autoloader = $autoloader;
 		} else {
-			$this->autoloader = new WCS_Core_Autoloader( $this->get_subscriptions_core_directory() );
+			$this->autoloader = new WCS_Core_Autoloader( $this->get_subscriptions_core_directory()  );
```

Comment:
**Nitpick:** Can probably revert this change 😄

Reply:
haha whoops 🤦 
I was testing something the other day with husky and completely forgot about this commit 😂